modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -147,6 +150,13 @@ class PersistentActor[S, M, R, E](
   import PersistentActor._
   import context.dispatcher
 
+  //FIXME: temporary fix to support switch between akka and existing shard allocation strategy
+  def aggregateId: String = aggregateIdOpt.getOrElse(self.path.name)
```

Comment:
Actually this isn't a bad assumption - in almost any real-world case `aggregateIdOpt` should be `Some(aggId)` and the actor itself is named by the aggregateId